apps/helpthechickens.com/src/app/page.tsx

```diff
@@ -61,21 +63,247 @@ const Hero = () => {
   );
 };
 
-const PetitionSection = () => {
+function PetitionSection() {
   return (
     <Section
       className="gap-12 items-center bg-slate-200 xl:rounded-lg py-12 md:px-16"
       id="petition-section"
     >
       <h2 className="font-semibold text-xl uppercase self-start text-center md:text-left w-full">
-        Ask Petaluma City Council to shut down Perdue's slaughterhouse
+        Ask Petaluma City Council to shut down Perdue&apos;s slaughterhouse
       </h2>
       <EmailPetition defaultMessage={DEFAULT_MESSAGE} onSubmit={onSubmit} />
     </Section>
   );
 };
 
-const Footer = () => {
+function MoreBackgroundSection() {
+  function ExposingPerdueYouTubeVideo() {
+    return (
+      <iframe
```

Comment:
does this have nice responsiveness? i feel like i've had to do a couple of tricks to make youtube videos behave nicely in the past.

Reply:
fixed!